geode-core/src/test/java/org/apache/geode/internal/cache/entries/AbstractRegionEntryTest.java

```diff
@@ -357,29 +357,23 @@ public void resolverShouldHandleConflictEventsFromDifferentDS() {
 
     private Object value;
     private VersionTag tag;
-    private long timeStamp = 0;
     private int dsId;
 
     protected TestableRegionEntry(RegionEntryContext context, Object value) {
       super(context, value);
     }
 
     @Override
-    public void setVersionTimeStamp(long timeStamp) {
-      this.timeStamp = timeStamp;
-    }
+    public void setVersionTimeStamp(long timeStamp) {}
```

Comment:
why do you still have changes to this test? Why not just checkout the version you did not modify so that this pull request will have no diffs on this file?